tests/vmi_configuration_test.go

```diff
@@ -486,7 +486,7 @@ var _ = Describe("Configurations", func() {
 
 				res, err := expecter.ExpectBatch([]expect.Batcher{
 					&expect.BSnd{S: "free -m | grep Mem: | tr -s ' ' | cut -d' ' -f2\n"},
-					&expect.BExp{R: "105"},
+					&expect.BExp{R: "104"},
```

Comment:
This is just looking for trouble... What is this even testing?
Sorry, again, not your doing, but I don't believe RAM usage in Linux is deterministic.
Could we maybe switch to a reasonable range to limit potential flakyness?

Reply:
It `grep`s total memory. That's deterministic or? The reason why this dropped is memballoon which is added by default.

Reply:
My bad, assumed it was used memory. Thanks!

Reply:
Comment on top of the command wouldn't hurt me either :)